### ISSUE TITLE
it says it can't create the virtual disk

### ISSUE THREAD

**Issue by tinyttvt** [foxlet/macOS-Simple-KVM#259]

**Comment by asgmeonerandom** [comment]
add info

**Comment by tinyttvt** [comment]
How

**Comment by olli1976** [comment]
Hello tinyttvt, what exactly is your problem. Post the error message and we can help you. Your system data is also helpful.

**Comment by tinyttvt** [comment]
I figured out the issue. It didn’t fully unpack the the dmg so I couldn’t run it

**Comment by olli1976** [comment]
Hello, can you please describe your solution of the problem exactly ? So it can help other users.

**Comment by tinyttvt** [comment]
When converting the dmg to img it didn’t fully unpack it. My solution was to do a fresh install and it became fine

**Comment by asgmeonerandom** [comment]
buy a mac, you barely know how to describe an issue and it will be impossible to help you
tl;dr: buy a mac

**Comment by SuperKaninchen** [comment]
> buy a mac, you barely know how to describe an issue and it will be impossible to help you

please remember that this is the first issue tinyttvt has opened on GitHub. Of course, the lack of information makes it almost impossible to try and resolve the issue, and I fully understand that.

> tl;dr: buy a mac

As far as I know, the reason this project was brought to life was so people wouldn't have to buy a mac.
Let's just all work together to resolve any issue :)

**Comment by SuperKaninchen** [comment]
> When converting the dmg to img it didn’t fully unpack it. My solution was to do a fresh install and it became fine

Although your issue has been resolved, please still post any error messages and/or steps you took to resolve the issue, as some people may have the exact same issue and are trying to look for a solution :)

**Comment by asgmeonerandom** [comment]
> > buy a mac, you barely know how to describe an issue and it will be impossible to help you
> 
> please remember that this is the first issue tinyttvt has opened on GitHub. Of course, the lack of information makes it almost impossible to try and resolve the issue, and I fully understand that.
> 
> > tl;dr: buy a mac
> 
> As far as I know, the reason this project was brought to life was so people wouldn't have to buy a mac.
> Let's just all work together to resolve any issue :)

How is it legitimate? You can't just download a OS, it costs money. a years of development that in the end u are using the mobile OS

**Comment by tinyttvt** [comment]
It’s legit. It was my first issue I’ve got the van working and everything.

**Comment by tinyttvt** [comment]
> > When converting the dmg to img it didn’t fully unpack it. My solution was to do a fresh install and it became fine
> 
> 
> 
> Although your issue has been resolved, please still post any error messages and/or steps you took to resolve the issue, as some people may have the exact same issue and are trying to look for a solution :)

The steps I took to resolve it were to do a fresh install but I was booting off of a live usb so that might’ve been my issue when writing the issue. I hope that helps.

**Comment by tinyttvt** [comment]
> > > buy a mac, you barely know how to describe an issue and it will be impossible to help you
> 
> > 
> 
> > please remember that this is the first issue tinyttvt has opened on GitHub. Of course, the lack of information makes it almost impossible to try and resolve the issue, and I fully understand that.
> 
> > 
> 
> > > tl;dr: buy a mac
> 
> > 
> 
> > As far as I know, the reason this project was brought to life was so people wouldn't have to buy a mac.
> 
> > Let's just all work together to resolve any issue :)
> 
> 
> 
> How is it legitimate? You can't just download a OS, it costs money. a years of development that in the end u are using the mobile OS in big desktop

I mean how is it not legitimate. The only reason why I tried this on my laptop was because I wanted to test it.

**Comment by tinyttvt** [comment]
> buy a mac, you barely know how to describe an issue and it will be impossible to help you
 This is my first issue so sorry if I can’t describe it well but I’m trying. 
> tl;dr: buy a mac
Sorry isn’t the exact reason for this project is to not buy a Mac for macOS.

**Comment by tinyttvt** [comment]
I’m going to close this now
